runelite-client/src/main/java/net/runelite/client/plugins/microbot/zerozero/bluedragons/BlueDragonsConfig.java

```diff
@@ -0,0 +1,95 @@
+package net.runelite.client.plugins.microbot.zerozero.bluedragons;
+
+import net.runelite.client.config.*;
+import net.runelite.client.plugins.microbot.util.misc.Rs2Food;
+
+@ConfigGroup("bluedragons")
+public interface BlueDragonsConfig extends Config {
+
+    @ConfigItem(
+            keyName = "How To",
+            name = "How to use",
+            description = "How to use the blue dragon plugin",
+            position = 0
+    )
+    default String howTo()
+    {
+        return "Start with falador runes/teletab and correct number of food from config";
```

Comment:
There is a @ConfigInformation annotation you can use to display the description for your script, this can never be overwritten and will be displayed at the top of your script. The content of the annotation is html, so you can add some styling in it etc..

Example of driftnet plugin:

@ConfigInformation("Start this script at the driftet fishing area. <br /> Make sure to have driftnet in your inventory or driftnet stored with anetta.")

Reply:
All done